main.go

```diff
@@ -65,5 +65,5 @@ func main() {
 		console.Fatalln("Go runtime version check failed.", err)
 	}
 
-	minio.Main(os.Args, os.Exit)
+	minio.Main(os.Args)
```

Comment:
> Are you talking about functional tests?

Yes.

The goal is to be able to test minio.Main() function. If we let this latter calls os.Exit(), `go test` will just stop and report success though it should run all of our written tests.